gradle/publishing.gradle

```diff
@@ -18,7 +18,6 @@ publishing {
             artifact sourcesJar
             artifact testsJar
             artifact javadocJar
-            artifact shadowJar
```

Comment:
@martinda why did we remove this? Will the shadow jar still got published if we remove it?

And apologies for the late responses ... just getting back into the swing of things now from the holidays.

Reply:
Hello @cdancy . It is automatically added since version [shadow version 6.1.0](https://imperceptiblethoughts.com/shadow/changes/#v6-1-0-2020-10-05) [PR #357]